Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WinForms/WindowsXamlHostBaseExt.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Win32.UI.Interop;
+using Microsoft.Toolkit.Win32.UI.Interop.WinForms;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WinForms
+{
+    public abstract class WindowsXamlHostBaseExt : WindowsXamlHostBase
```

Comment:
The name here is terrible. I left another comment on the WPF counterpart of this. After seeing both I see that each has their own implementation for each UI framework. `WindowsFormsXamlHostBase`?